### PR TITLE
Add SDCoreAccess and SDFabricAccess groups

### PR DESCRIPTION
**Pull request by zdw**

### CODE REVIEW

File: functions/conf/crowd_to_github_groups.json [hunk 1]
```diff
@@ -51,6 +51,26 @@
       "team": "SDRANAccess"
     }
   ],
+  "SDCoreAccess" : [
+    {
+      "githubOrg": "omec-project",
+      "team": "SDRANAccess"
```
Comment (ccascone) [inline functions/conf/crowd_to_github_groups.json:57]: Should this be `SDCoreAccess`? The `SDRANAccess` team is already synchronized with `omec-project` (see above).
```suggestion
      "team": "SDCoreAccess"
```

File: functions/conf/crowd_to_github_groups.json [hunk 2]
```diff
@@ -51,6 +51,26 @@
...
+    },
+    {
+      "githubOrg": "onosproject",
+      "team": "SDRANAccess"
```
Comment (ccascone) [inline functions/conf/crowd_to_github_groups.json:61]: Same here

File: functions/conf/crowd_to_github_groups.json [hunk 3]
```diff
@@ -51,6 +51,26 @@
...
+    }
+  ],
+  "SDFabricAccess" : [
+    {
+      "githubOrg": "omec-project",
+      "team": "SDRANAccess"
```
Comment (ccascone) [inline functions/conf/crowd_to_github_groups.json:67]: Same here, should this be `SDFabricAccess`?

File: functions/conf/crowd_to_github_groups.json [hunk 4]
```diff
@@ -51,6 +51,26 @@
...
+    },
+    {
+      "githubOrg": "onosproject",
+      "team": "SDRANAccess"
```
Comment (ccascone) [inline functions/conf/crowd_to_github_groups.json:71]: Same here

File: functions/conf/crowd_to_github_groups.json (hunk 1 above)
Comment (zdw) [reply functions/conf/crowd_to_github_groups.json]: 🤦 Fixed.

**Review by teone — APPROVED** [review]

**Pull request merged**